src/Microsoft.NetFramework.Analyzers/Core/SerializationRulesDiagnosticAnalyzer.cs

```diff
@@ -224,6 +224,12 @@ public void AnalyzeSymbol(SymbolAnalysisContext context)
                         namedTypeSymbol.GetMembers().OfType<IFieldSymbol>().Where(m => !IsSerializable(m.Type));
                     foreach (IFieldSymbol field in nonSerializableFields)
                     {
+                        // Only process instance fields
```

Comment:
📝 I normally place comments like this immediately inside the braces. This allows *consistent* placement of documenting comments across `if` statements with a variety of `if`/`else if`/`else` forms.

Reply:
I followed the pattern used throughout this analyzer.